packages/mailchimp/configuration-schema.json

```diff
@@ -5,23 +5,21 @@
             "title": "Data Center",
             "type": "string",
             "description": "Mailchimp Data Center for your account",
-            "minLength": 3,
+            "minLength": 1,
```

Comment:
@elias-ba i am curious to know why `1`, because from my understanding mailchimp data centers have at least `3` characters going up, For example 
`uk8` 3 characters, `us11` 4 characters